src/main/scala/com/databricks/labs/overwatch/pipeline/Module.scala

```diff
@@ -351,6 +351,7 @@ class Module(
   @throws(classOf[IllegalArgumentException])
   def execute(_etlDefinition: () => ETLDefinition): ModuleStatusReport = {
 //    _etlDefinition: ETLDefinition
+    spark.sessionState.catalogManager.setCurrentCatalog(config.etlCatalogName)
```

Comment:
Since you set this in the Initializer, why do you keep setting it throughout the code?

Reply:
After each layer completion it will set to default hive_metastore